_benchmark/reference/telemetry.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: default
+title: Telemetry devices
+nav_order: 45
+has_children: true
+---
+
+# Telemetry devices
+
+Telemetry devices give you additional insights on benchmark results. To see a list of telemetry devices available, use `opensearch-benchmark list telemetry`. 
+
+All telemetry devices with a `-stats` can be used with clusters not provisioned by OpenSearch Benchmark. These devices are referred to as **Runtime level telemetry devices**. Alternatively, **Setup level telemetry devices** encompass devices that can only be used when OpenSearch Benchmark provisions a cluster. 
+
+This page lists the telemetry devices supported by OpenSearch Benchmark.
+
+## jfr
+
+The `jfr` telemetry device enables the [Java Flight Recorder (JFR)](https://docs.oracle.com/javacomponents/jmc-5-5/jfr-runtime-guide/index.html) on the benchmark candidate. Up to JDK 11, JFR ships only with Oracle JDK. OpenSearch Benchmark assumes that Oracle JDK is used for benchmarking. If you run benchmarks on JDK 11 or later, [Java flight recorder](https://jdk.java.net/jmc/) is also available on OpenJDK.
+
+To enable `jfr`, invoke Workload with `opensearch-benchmark workload --workload=pmc --telemetry jfr`. `jfr` will then write a flight recording file which can be opened in Java Mission Control. OpenSearch Benchmark prints the location of the flight recording file on the command line.
+
+The `jfr` devices supports the following parameters:
+
+
+- `recording-template`: The name of a custom flight recording template. It is up to you to correctly install these recording templates on each target machine. If none is specified, the default recording template of Java flight recorder is used.
+- `jfr-delay`: (Optional) The length of time to wait before starting to record.  
+- `jfr-duration`: (Optional) The length of time to record. 
+
+## jit
+
+The `jit` telemetry device enables JIT compiler logs for the benchmark candidate. If the HotSpot disassembler library is available, the logs will also contain the disassembled JIT compiler output which can be used for low-level analysis.
```

Comment:
Should "benchmark" be OpenSearch Benchmark?"

Reply:
See previous comment.